src/main/java/PlayerData.java

```diff
@@ -0,0 +1,150 @@
+/** relevant imports*/
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+/**
+ * The PlayerData class
+ */
+public class PlayerData implements Serializable {
+    /** Stores the health of the PlayerData object */
+    private int health;
```

Comment:
Since we are restoring the player's health at the end of every combat win, should we change the name of the attribute to be maxHealth?